.env

```diff
@@ -1 +1 @@
-REACT_APP_API_URI="https://crosproxy001.herokuapp.com/payitapi.herokuapp.com:443/api"
+REACT_APP_API_URI="https://crosproxy001.herokuapp.com/payingitforward.re-coded.com:443/api"
```

Comment:
Doesn't https://payingitforward.re-coded.com/api work ? Why do we need the crosproxy.herokuapp here ?